internal/config/constants.go

```diff
@@ -0,0 +1,21 @@
+// Copyright 2025 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package config
+
+const (
+	// GeneratorInputDir is the default directory to store files that generator
+	// needs to regenerate libraries from an empty directory.
+	GeneratorInputDir string = "generator-input"
```

Comment:
instead of having a separate file with one constant, just put it in internal/config/config.go